CHANGELOG.md

```diff
@@ -1,5 +1,82 @@
 # [Versions](https://mui.com/versions/)
 
+## 5.13.6
+
+_Jun 21, 2023_
+
+A big thanks to the 25 contributors who made this release possible. Here are some highlights ✨:
+
+- 💫 Added `Slider` component using the new Material You design language (#37520) @DiegoAndai.
+- 📚 Added  overview docs page showcasing all Base UI components (#37536) @danilo-leal
```

Comment:
Perfect, let's just update these two items with links to the slider demo & the example directory.